src/transactions/transactions.service.ts

```diff
@@ -62,13 +62,17 @@ export class TransactionsService {
     this.legitimacySecret = config.legitimacySecret;
   }
 
+  public async getSigningAccount(signer: string): Promise<string> {
```

Comment:
```suggestion
  private async getSigningAccount(signer: string): Promise<string> {
```
I like this method, but it should be private:

If someone needs the lookup they should import signingService directly.